exercises/patterns/printHollowSquare/printHelpers.js

```diff
@@ -0,0 +1,42 @@
+let process = require('process');
```

Comment:
There shouldn't have been any need to copy this file multiple times, but if something wasn't working until you did let me know.